{
  "debianbug": 700948,
  "description": "The XML libraries for Python 3.4, 3.3, 3.2, 3.1, 2.7, and 2.6, as used in OpenStack Keystone Essex, Folsom, and Grizzly; Compute (Nova) Essex and Folsom; Cinder Folsom; Django; and possibly other products allow remote attackers to cause a denial of service (resource consumption and crash) via an XML Entity Expansion (XEE) attack.",
  "releases": {
    "bullseye": {
      "fixed_version": "2012.1.1-13",
      "repositories": {
        "bullseye": "2:15.0.0-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "buster": {
      "fixed_version": "2012.1.1-13",
      "repositories": {
        "buster": "2:14.0.1-2"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "jessie": {
      "fixed_version": "2012.1.1-13",
      "repositories": {
        "jessie": "2014.1.3-6"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "sid": {
      "fixed_version": "2012.1.1-13",
      "repositories": {
        "sid": "2:15.0.0-4"
      },
      "status": "resolved",
      "urgency": "medium**"
    },
    "stretch": {
      "fixed_version": "2012.1.1-13",
      "repositories": {
        "stretch": "2:10.0.0-9+deb9u1",
        "stretch-security": "2:10.0.0-9+deb9u1"
      },
      "status": "resolved",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}